pagopa-ecommerce-cdc-service/ecommerce-cdc-service.env

```diff
@@ -54,7 +54,7 @@ REDIS_PASSWORD=redis_password
 REDIS_SSL_ENABLED=false
 
 # Transaction view configuration
-ECOMMERCE_TRANSACTION_VIEW_COLLECTION_NAME=transaction-view-cdc
+ECOMMERCE_TRANSACTION_VIEW_COLLECTION_NAME=transactions-view
```

Comment:
this name should be overridden during cdc integration tests execution only for the moment